gulpfile.js

```diff
@@ -97,15 +98,6 @@ gulp.task('app-partials', function () {
 gulp.task('app-geny-window', function () {
     const version = getVersion(); // Fetch the version from package.json
 
-    gulp.src(['./example/geny-window.html'])
-        .pipe(
-            replace(
-                /https:\/\/cdn\.jsdelivr\.net\/npm\/@genymotion\/device-web-player@[\d\.]+/g,
-                `https://cdn.jsdelivr.net/npm/@genymotion/device-web-player@${version}`,
-            ),
-        )
-        .pipe(gulp.dest('./example/'));
-
```

Comment:
I'm still not sure why you want to get rid of this replace step?

Reply:
in line below we parse all example folder, so this code is executed twice